src/ai/backend/manager/registry.py

```diff
@@ -484,13 +484,13 @@ def __init__(self, config_server, dbpool,
                     # if not belong to the execution kernel.
                     continue
                 matched_mounts.add(item['name'])
-                determined_mounts.append((
+                determined_mounts.append({
                     item['name'],
                     item['host'],
                     item['id'].hex,
                     item['permission'].value,
                     item['unmanaged_path'] if item['unmanaged_path'] else ''
-                ))
+                })
```

Comment:
Why is this changed to a list of sets from a list of tuples?

Reply:
Not an intended type change; looks like a mistake while editing other codes. I'll fix it at next commit.